elasticsearch_dsl/__init__.py

```diff
@@ -159,3 +161,31 @@
     "token_filter",
     "tokenizer",
 ]
+
+
+try:
+    # Asyncio only supported in Python 3.6+
+    if sys.version_info < (3, 6):
+        raise ImportError
+
+    from elasticsearch_dsl._async.faceted_search import AsyncFacetedSearch
+    from elasticsearch_dsl._async.index import AsyncIndex, AsyncIndexTemplate
+    from elasticsearch_dsl._async.mapping import AsyncMapping
+    from elasticsearch_dsl._async.search import AsyncMultiSearch, AsyncSearch
+    from elasticsearch_dsl._async.update_by_query import AsyncUpdateByQuery
+
+    __all__ = sorted(
+        __all__
+        + [
+            "AsyncFacetedSearch",
+            "AsyncIndex",
+            "AsyncIndexTemplate",
+            "AsyncMapping",
+            "AsyncMultiSearch",
+            "AsyncSearch",
+            "AsyncUpdateByQuery",
+        ]
+    )
+
+except (ImportError, SyntaxError):
```

Comment:
It looks like the `__all__ = sorted()` line doesn't throw the `ImportError` or `SyntaxError`. My preference is to keep `try` blocks as small as possible. Can this be rewritten like so?

    try:
        from elasticsearch_dsl._async.faceted_search import AsyncFacetedSearch
        ...
        from elasticsearch_dsl._async.update_by_query import AsyncUpdateByQuery
    except (ImportError, SyntaxError):
        pass
    else:
        __all__ = sorted()